.github/workflows/bson-ruby.yml

```diff
@@ -20,15 +20,13 @@ jobs:
       fail-fast: false
       matrix:
         os: [ ubuntu, macos, windows ]
-        ruby: [ 2.3, 2.4, 2.5, 2.6, 2.7, head ]
-        include:
-          - { os: windows , ruby: mingw }
```

Comment:
Why was windows removed please?

Reply:
Stable rubies on windows are still included. I was not sure whether we support `mingw` version. Do we need it?

Reply:
should be mingw64, and yes, please include this! :)

Reply:
(It might be `x64_mingw` actually.)

Reply:
However this is the Ruby "platform", not the ruby "version"

Reply:
mingw is back, as well as ruby head for macos and linux